client/asset/eth/contractor_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"decred.org/dcrdex/client/asset"
 	"decred.org/dcrdex/dex/encode"
-	dexeth "decred.org/dcrdex/dex/networks/eth"
+	"decred.org/dcrdex/dex/networks/eth/swap"
```

Comment:
Thinking about multiple versions, I think we'll end up like:

```
swapv0 "decred.org/dcrdex/dex/networks/eth/swap/v0"
swapv1 "decred.org/dcrdex/dex/networks/eth/swap/v1"
```

Reply:
That makes sense.

Reply:
Moved it to a v0 package.